apps/src/studio/studio.js

```diff
@@ -1040,6 +1040,10 @@ Studio.onTick = function() {
     Studio.succeededTime = currentTime;
   }
 
+  if (!animationOnlyFrame) {
+    Studio.executeQueue('whenTouchGoal');
```

Comment:
i never pondered multiple queues before.. we execute them all separately when it's their time?